src/main/modules/nunjucks/index.ts

```diff
@@ -29,15 +29,16 @@ export class Nunjucks {
                 express: app,
             }
         );
-        const addFilters = require('./njkFilters');
-        addFilters(env);
+        /* eslint-disable-next-line @typescript-eslint/no-require-imports */
+        const njkFilters = require('./njkFilters');
```

Comment:
Are we not able to do the following?

import { createFilters } from './njkFilters';
createFilters(env);

Reply:
Unfortunately not :( 

That kind of import needs to be at the top level and not within a a method.

You can have an import like this at method level:

const { createFilters} = await import './njkFilters', 

however due to the await it then needs an async. However unfortunately with this file being called from the app.ts, you cannot add an async unless you turn the application into a module